uai2017/python/superuser-init.py

```diff
@@ -0,0 +1,338 @@
+            readers=['UAI.org/2017/pcs','UAI.org/2017'],
+            writers=['UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/pcs'],
+            signatories=['UAI.org/2017/conference/reviewers-invited'],
+            members=[])
+        groups.append(uai2017reviewersinvited)
+
+    if overwrite_allowed('UAI.org/2017/conference/reviewers-emailed'):
+        uai2017reviewersemailed = Group('UAI.org/2017/conference/reviewers-emailed',
+            readers=['UAI.org/2017/pcs','UAI.org/2017'],
+            writers=['UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/pcs'],
+            signatories=['UAI.org/2017/conference/reviewers-emailed'],
+            members=[])
+        groups.append(uai2017reviewersemailed)
+
+
+    if overwrite_allowed('UAI.org/2017/conference/reviewers'):
+        uai2017reviewers = Group('UAI.org/2017/conference/reviewers',
+            readers=['everyone'],
+            writers=['UAI.org/2017/conference','UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/conference'],
+            signatories=['UAI.org/2017/conference/reviewers'],
+            members=[])
+        groups.append(uai2017reviewers)
+
+
+    if overwrite_allowed('UAI.org/2017/conference/reviewers-declined'):
+        uai2017reviewersdeclined = Group('UAI.org/2017/conference/reviewers-declined',
+            readers=['everyone'],
+            writers=['UAI.org/2017/conference','UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/conference'],
+            signatories=['UAI.org/2017/conference/reviewers'],
+            members=[])
+        groups.append(uai2017reviewersdeclined)
+
+
+    ## Post the groups
+    for g in groups:
+        print "Posting group: ",g.id
+        openreview.post_group(g)
+
+    openreview.post_group(openreview.get_group('host').add_member('UAI.org/2017'))
+
+
+
+
+    #########################
+    ##  SETUP INVITATIONS  ##
+    ######################### 
+
+
+    ## Create the submission invitation
+    reply = {
+        'forum': None,
+        'replyto': None,
+        'readers': {
+            'description': 'The users who will be allowed to read the above content.',
+            'values': ['everyone']
+        },
+        'signatures': {
+            'description': 'How your identity will be displayed with the above content.',
+            'values-regex': '~.*'
+        },
+        'writers': {
+            'values-regex': '~.*'
+        },
+        'content': {
+            'title': {
+                'description': 'Title of paper.',
+                'order': 1,
+                'value-regex': '.{1,100}',
+                'required':True
+            },
+            'authors': {
+                'description': 'Comma separated list of author names, as they appear in the paper.',
+                'order': 2,
+                'value-regex': '[^,\\n]+(,[^,\\n]+)*',
+                'required':True
+            },
+            'author_emails': {
```

Comment:
author_emails were renamed to author_ids: https://github.com/iesl/openreview-scripts/blob/master/iclr2017/python/superuser-init.py#L221

author and author_ids are values-regex instead of value_regex. 

I think @agankur21 need to update the last changes of iclr2017, what do you think @mspector